tests/test_warc_to_zim.py

```diff
@@ -848,3 +853,29 @@ def test_content_resource_types(self, tmp_path):
         ]:
             res = self.get_article(zim_output, js_file)
             assert b"wombat" in res  # simple check that rewriting has been done
+
+    def test_content_encoding_aliases(self, tmp_path):
+        zim_output = "tests_en_qsl.net-encoding-alias.zim"
+
+        main(
+            [
+                os.path.join(
```

Comment:
Not `TEST_DATA_SPECIAL_DIR` ? Why can't it be in test_all_warcs_root_dir?

Reply:
It cannot be processed like other TEST_DATA_DIR warcs since it needs special encoding aliases to be used in --encoding-aliases. Comment added in https://github.com/openzim/warc2zim/pull/347/commits/ddbf74e81b3b16eca8ada90326af187759be2b6c

Reply:
(which is now https://github.com/openzim/warc2zim/pull/347/commits/bcc71e66d77c86a2ee6560405668051cbc019d4e=